pkg/svcat/service-catalog/binding_test.go

```diff
@@ -402,12 +408,22 @@ var _ = Describe("Binding", func() {
 
 			Expect(actions[0].Matches("list", "servicebindings")).To(BeTrue())
 			Expect(actions[0].(testing.ListActionImpl).Namespace).To(Equal(si.Namespace))
-			Expect(actions[1].Matches("get", "servicebindings")).To(BeTrue())
-			Expect(actions[1].(testing.GetActionImpl).Name).To(Equal(sb.Name))
-			Expect(actions[1].(testing.GetActionImpl).Namespace).To(Equal(sb.Namespace))
-			Expect(actions[2].Matches("update", "servicebindings")).To(BeTrue())
-			Expect(actions[2].(testing.UpdateActionImpl).Object.(*v1beta1.ServiceBinding).ObjectMeta.Name).To(Equal(sb.ObjectMeta.Name))
-			Expect(actions[2].(testing.UpdateActionImpl).Object.(*v1beta1.ServiceBinding).ObjectMeta.Namespace).To(Equal(sb.ObjectMeta.Namespace))
+			// For the remaining actions, one should be get and the other one should be update
```

Comment:
I think there is no need to use the pattern from the previous change. Correct me if I'm wrong, but always the "get" action will be done before the "update". The problem occurs when there are two concurrent execution, but there is only one. I think the change is not necessary, but of course the test is green.

Reply:
@piotrmiskiewicz Yeah you are right. Here's only one service binding to be removed and therefore no concurrent issue comparing to the test for `RemoveFinalizerForBindings `.